### PR TITLE
docs: update android integration guide

### PR DESCRIPTION
**Pull request by xiaoyijun**
update android integration guide

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx [hunk 1]
```diff
@@ -49,12 +49,10 @@ dependencies {
 
 ## 步骤 2： 配置
 
-### 配置 Redirect URI
-
-Redirect URI 指定了应用用来接受授权结果的端口，Logto Android SDK 内部实现了该重定向的功能
-
 ### 在管理员控制台中配置 Redirect URI
 
+Redirect URI 指定了应用用来接受授权结果的端口.
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:54]: - receive → 接收?
- endpoint → 端点
- . → 。
```suggestion
Redirect URI 指定了应用用来接收授权结果的端点。
```

File: docs/docs/recipes/integrate-logto/android.mdx [hunk 2]
```diff
@@ -49,12 +49,10 @@ dependencies {
 
 ## Configure
 
-### Configure Redirect URI
-
-The `Redirect URI` indicates the endpoint that the application used to receive authorization results, which is implemented by the Logto Android SDK internally.
-
 ### Configure Redirect URI in Admin Console
 
+The `Redirect URI` indicates the endpoint that the application used to receive authorization results.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/android.mdx:54]: Improved by Grammarly
```suggestion
The `Redirect URI` indicates the application's endpoint to receive authorization results.
```

File: docs/docs/recipes/integrate-logto/android.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Android (Kotlin)
+sidebar_label: Android
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/android.mdx:2]: maybe better keep ` (Kotlin)`?

Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:2]: Why? Java developers can also use this documentation.

**Review by IceHe — APPROVED** [review]
LGTM. Better wait for another approval.

File: docs/docs/recipes/integrate-logto/android.mdx [hunk 4]
```diff
@@ -49,12 +49,10 @@ dependencies {
 
 ## Configure
 
-### Configure Redirect URI
-
-The `Redirect URI` indicates the endpoint that the application used to receive authorization results, which is implemented by the Logto Android SDK internally.
-
 ### Configure Redirect URI in Admin Console
```
Comment (gao-sun) [inline docs/docs/recipes/integrate-logto/android.mdx:52]: you need put an admonition here like other recipes

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx [hunk 5]
```diff
@@ -49,12 +49,10 @@ dependencies {
 
 ## 步骤 2： 配置
 
-### 配置 Redirect URI
-
-Redirect URI 指定了应用用来接受授权结果的端口，Logto Android SDK 内部实现了该重定向的功能
-
 ### 在管理员控制台中配置 Redirect URI
 
+Redirect URI 指定了应用用来接收授权结果的端点。
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:54]: we don't need to translate endpoint

File: docs/docs/recipes/integrate-logto/android.mdx (hunk 4 above)
Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:53]: updated.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx (hunk 5 above)
Comment (xiaoyijun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx]: updated.

File: docs/docs/recipes/integrate-logto/android.mdx (hunk 4 above)
Comment (gao-sun) [reply docs/docs/recipes/integrate-logto/android.mdx:53]: well i was think about the import of `<AppNote />` and `<SignInNote />` etc. would you like to go through the code and follow the latest integration guide format?

Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:58]: updated.

File: docs/docs/recipes/integrate-logto/android.mdx [hunk 6]
```diff
@@ -1,17 +1,29 @@
...
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import SignInNote from './fragments/_sign-in-note.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import redirectUriFigure from './assets/android-redirect-uri.png';
 
-# Android: Integrate `io.logto.sdk:android`
+
+# Android: Integrate Logto Android SDK
```
Comment (demonzoo) [inline docs/docs/recipes/integrate-logto/android.mdx:14]: Fix these lint issues and we are good to move on

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx [hunk 7]
```diff
@@ -206,7 +224,7 @@ logtoClient.fetchUserInfo((logtoException, userInfoResponse) -> {
 
 </Tabs>
 
-## 步骤 4： 登出
+## 登出
```
Comment (demonzoo) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:227]: 退出登录

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx [hunk 8]
```diff
@@ -206,7 +224,7 @@ logtoClient.fetchUserInfo((logtoException, userInfoResponse) -> {
...
 
 ### 执行登出
```
Comment (demonzoo) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx:229]: 执行退出登录？

File: docs/docs/recipes/integrate-logto/android.mdx (hunk 6 above)
Comment (xiaoyijun) [reply docs/docs/recipes/integrate-logto/android.mdx:13]: fixed.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx (hunk 7 above)
Comment (xiaoyijun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx]: updated.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx (hunk 8 above)
Comment (xiaoyijun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/android.mdx]: removed.

**Review by demonzoo — APPROVED** [review]
LGTM

**Pull request merged**